vuepress/docs/next/tutorials/devops/installation/azure-kubernetes-service/azure-install.md

```diff
@@ -27,127 +27,120 @@ If you're already comfortable setting up an AKS cluster and installing nginx the
 2. Select `Connect`
-3. Select `Bash`
+3. Select `Open Cloud Shell`
+   - With a new Azure account you may see a warning: `You have no storage mounted`. Follow the instructions to create a new storage account.
 4. Run the first two commands (e.g. `az account set...` and `az aks get-credentials...` to connect to your cluster. This should only be needed the first time you run the Azure Cloud Shell.
-    - The Cloud Shell times out after 20 minutes of inactivity.
-    - The following instructions assume you'll use the Azure Cloud Shell but you can also run the commands in a local environment if you have `kubectl`
-5. Deploy nginx with the commands below. See [nginx instructions](https://docs.microsoft.com/en-us/azure/aks/ingress-basic) for more details.
-
-```
-kubectl create namespace ingress-basic
+    - The following instructions assume you'll use the Azure Cloud Shell but you can also run the commands in a local environment via `kubectl`
+5. Add the NGINX controller to enable access to the cluster
+``` sh
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/controller-v1.0.5/deploy/static/provider/cloud/deploy.yaml 
 ```
+   See <https://kubernetes.github.io/ingress-nginx/deploy/#azure> for more information.
 
+6. Get the external IP address for your ingress controller.
+``` sh
+kubectl get services -n ingress-nginx
 ```
-helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx
-helm repo add stable https://kubernetes-charts.storage.googleapis.com/
-helm repo update
-```
-
-```
-helm install nginx-ingress ingress-nginx/ingress-nginx \
-    --namespace ingress-basic \
-    --set controller.replicaCount=2 \
-    --set controller.nodeSelector."beta\.kubernetes\.io/os"=linux \
-  --set defaultBackend.nodeSelector."beta\.kubernetes\.io/os"=linux
-```
-
-
-6. Get the external IP address for your ingress controller. Record the value of EXTERNAL-IP for `nginx-ingress-controller` from the command below.
-
-```
-kubectl get service -n ingress-basic
+   The output should be similar to this:
+``` 
+NAME                      TYPE          CLUSTER-IP    EXTERNAL-IP                        
+ingress-nginx-controller  LoadBalancer  10.0.28.197   20.120.54.243
 ```
+   Record the value of the EXTERNAL-IP.
 
 ### Verify the NGINX Ingress Install
-We recommend setting up a test application so you can easily verify the ingress is working in your cluster. See [this page](../google-cloud-platform/gke-install#verify-the-nginx-ingress-install) for those steps. You can use either Azure Cloud Shell or your local `kubectl`.
+We recommend verifying nginx is working correctly. The simplest option is to access the EXTERNAL-IP in your browser. You should get a `404 Not Found` nginx error page. Alternatively you can set up a simple test application. See [this page](../google-cloud-platform/gke-install#verify-the-nginx-ingress-install) for those steps. You can use either Azure Cloud Shell or your local `kubectl`.
```

Comment:
is it nginx or NGINX?

Reply:
The community tends to use nginx, their official docs use NGINX now.  Didn't realized so will change to match, just for the AWS and Azure guides for now.